src/events/pollVotationResults.js

```diff
@@ -43,24 +44,25 @@ module.exports = {
 
       const channel = await client.channels.fetch(channelId);
 
-      if (channel) {
-        if (finalResult > 0) {
-          await Promise.all(
-            Array.from({ length: finalResult }).map(async () => {
-              await channel.send(`<@&${selectedTagId}> ${userMentioned}`);
-            })
-          );
-        } else {
-          await channel.send(
-            `The result was not valid or no points were awarded.`
-          );
-        }
-      } else {
-        await message.reply({
+      if (!channel) {
+        await channel.send(`The draw is not supported`);
```

Comment:
I think this message should not be here?

Reply:
in old code it was when embeds[0].fields.length was less or equal to 3, maybe it can be when finalResult is not found?
![image](https://github.com/user-attachments/assets/cec8e087-bdc1-4c0b-bf09-4d7f779f3b1c)